app/assets/stylesheets/home.scss

```diff
@@ -198,3 +198,9 @@ a .project:hover  {
 .group-icon {
   width: 100%;
 }
+
+#main-content {
+  @media (min-width: 320px ) {
```

Comment:
Dit gaat enkel gebeuren bij schermen kleiner dan 320px

Gebruikt van hier: https://stackoverflow.com/questions/40542754/most-recommended-css-media-query-for-mobile-phones-and-tablet-devices